echopype/tests/echodata/test_echodata.py

```diff
@@ -272,6 +275,70 @@ def test_repr_html(self, converted_zarr):
             "<pre>&lt;EchoData"
         ) and html_fallback.endswith("</pre>")
 
+    def test_setattr(self, converted_zarr):
+        sample_data = xr.Dataset({"x": [0, 0, 0]})
+        sample_data2 = xr.Dataset({"y": [0, 0, 0]})
+        ed = EchoData.from_file(converted_raw_path=converted_zarr)
+        current_ed_beam = ed.beam
+        current_ed_top = ed.top
+        ed.beam = sample_data
+        ed.top = sample_data2
+
+        assert ed.beam.equals(sample_data) is True
+        assert ed.beam.equals(ed['Sonar/Beam_group1']) is True
+        assert ed.beam.equals(current_ed_beam) is False
+
+        assert ed.top.equals(sample_data2) is True
+        assert ed.top.equals(ed['Top-level']) is True
```

Comment:
Do we need the `is True` here and on lines 287,288? I understand why 289 needs it.

Reply:
Yea. This ensures that we get `True`. Probably don't need it, but I like to be explicit 😄